src/us/kbase/workspace/kbase/KBaseWorkspaceConfig.java

```diff
@@ -795,6 +796,8 @@ public boolean equals(Object obj) {
 				return false;
 		} else if (!delegateTypeTarget.equals(other.delegateTypeTarget))
 			return false;
+		if (dontTrustXIPHeaders != other.dontTrustXIPHeaders)
```

Comment:
presumably `dontTrustXIPHeaders` can never be `null`, hence no null test here

Reply:
Right, it's a `boolean` so always either `true` or `false`